_inc/lib/admin-pages/class.jetpack-landing-page.php

```diff
@@ -286,7 +286,6 @@ function page_admin_scripts() {
 				'modules'           => Jetpack::get_translated_modules( array_values( Jetpack_Admin::init()->get_modules() ) ),
 				'currentVersion'    => JETPACK__VERSION,
 				'ajaxurl'           => admin_url( 'admin-ajax.php' ),
-				'jumpstart_modules' => $this->jumpstart_module_tag( 'Jumpstart' ),
```

Comment:
Jumpstart functionality? It's being done in `class.jetpack-react-page.php`.  this file is not needed at all